distro/configs/openmrs/frontend_assembly/spa-assemble-config.json

```diff
@@ -1,47 +1,46 @@
 {
   "coreVersion": "6.3.1-pre.3149",
   "frontendModules": {
-    "@ugandaemr/esm-data-visualizer-app": "next",
+    "@openmrs/esm-active-visits-app": "8.4.2",
+    "@openmrs/esm-appointments-app": "8.4.2",
+    "@openmrs/esm-billing-app": "1.0.2-pre.546",
     "@openmrs/esm-cohort-builder-app": "4.0.3",
-    "@openmrs/esm-home-app": "5.6.3",
-    "@openmrs/esm-system-admin-app": "4.1.0",
     "@openmrs/esm-devtools-app": "6.3.1-pre.3149",
-    "@openmrs/esm-patient-flags-app": "10.2.1-pre.8163",
+    "@openmrs/esm-dispensing-app": "1.8.0",
+    "@openmrs/esm-fast-data-entry-app": "1.3.0",
+    "@openmrs/esm-form-builder-app": "3.1.1-pre.1896",
+    "@openmrs/esm-form-engine-app": "11.1.1-pre.8699",
+    "@openmrs/esm-generic-patient-widgets-app": "11.1.1-pre.8699",
+    "@openmrs/esm-help-menu-app": "6.3.1-pre.3149",
+    "@openmrs/esm-home-app": "9.0.0",
+    "@openmrs/esm-implementer-tools-app": "6.3.1-pre.3149",
+    "@openmrs/esm-laboratory-app": "1.2.1-pre.685",
+    "@openmrs/esm-login-app": "6.3.1-pre.3149",
     "@openmrs/esm-openconceptlab-app": "4.1.0",
-    "@openmrs/esm-patient-search-app": "8.4.2",
-    "@openmrs/esm-primary-navigation-app": "6.3.1-pre.3149",
-    "@openmrs/esm-active-visits-app": "8.4.2",
-    "@openmrs/esm-patient-lists-app": "10.2.1-pre.8163",
+    "@openmrs/esm-patient-allergies-app": "11.1.1-pre.8699",
+    "@openmrs/esm-patient-attachments-app": "11.1.1-pre.8699",
+    "@openmrs/esm-patient-banner-app": "11.1.1-pre.8699",
+    "@openmrs/esm-patient-chart-app": "11.1.1-pre.8699",
+    "@openmrs/esm-patient-conditions-app": "11.1.1-pre.8699",
+    "@openmrs/esm-patient-flags-app": "11.1.1-pre.8699",
+    "@openmrs/esm-patient-forms-app": "11.1.1-pre.8699",
+    "@openmrs/esm-patient-immunizations-app": "11.1.1-pre.8699",
     "@openmrs/esm-patient-list-management-app": "8.4.2",
-    "@openmrs/esm-login-app": "6.3.1-pre.3149",
-    "@openmrs/esm-patient-banner-app": "10.2.1-pre.8163",
+    "@openmrs/esm-patient-lists-app": "11.1.1-pre.8699",
+    "@openmrs/esm-patient-medications-app": "11.1.1-pre.8699",
```

Comment:
Hello @pirupius ..  Are you still working on this ? Some merge conflicts , Btw why did we reach on point of using such version term.